Containerfile.redis

```diff
@@ -1,33 +1,17 @@
 # Use Red Hat Universal Base Image (UBI) 9 as the base image
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 
-# Install necessary dependencies (gcc, make, etc.) for building Redis from source
-RUN microdnf install -y gcc make wget tar gzip && \
-    microdnf clean all
+# Install necessary dependencies for downloading and installing Redis
+RUN microdnf install -y wget yum-utils logrotate systemd && \
+    wget https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/redis-7.0.12-1.module_el9+499+353c08ce.x86_64.rpm && \
+    rpm -i redis-7.0.12-1.module_el9+499+353c08ce.x86_64.rpm && \
+    rm -f redis-7.0.12-1.module_el9+499+353c08ce.x86_64.rpm
 
-# Set the Redis version
-ARG REDIS_VERSION=7.2.4
-
-# Download and extract Redis source code
-RUN mkdir -p /etc/redis && \
-    wget https://download.redis.io/releases/redis-${REDIS_VERSION}.tar.gz && \
-    tar xzf redis-${REDIS_VERSION}.tar.gz -C /etc/redis --strip-components=1 && \
-    rm redis-${REDIS_VERSION}.tar.gz && \
-    chmod 777 /var/run
-
-# Change working directory to the extracted Redis source code directory
+# Set working directory for Redis
 WORKDIR /etc/redis
 
-# Build Redis from source
-RUN make && \
-    make install
-
-# Set protected-mode to no and bind to all interfaces (0.0.0.0) in redis.conf
-RUN sed -i 's/^protected-mode yes/protected-mode no/' redis.conf
-RUN sed -i 's/^bind 127.0.0.1 -::1/bind 0.0.0.0/' redis.conf
+# Grant necessary permissions
+RUN chmod -R 777 /var/lib/redis /var/log/redis /etc/redis
```

Comment:
It would be better to move these to the RUN command above so as not to create an extra image layer. Also, the perms seem too permissive. On the cluster, with the restricted-v2 SCC uid is whatever but gid is 0, would 770 be enough?

Reply:
`/etc/redis` might be just `+r`, not `+w` nor `+x`

Reply:
Acknowledged. Combined these with the above RUN command and granted only the required perms.